src/app/components/Hero/Hero.jsx

```diff
@@ -16,10 +16,10 @@ class Hero extends React.Component {
         <div className="Hero__container nypl-full-width-wrapper">
           <div className="Hero__text nypl-column-three-quarters nypl-column-offset-one">
             <div className="Hero__text__HeroTitle">
-              <h3>{this.props.heroData.category}</h3>
-              <p className="Hero__text__HeroTitle__des">
+              {this.props.heroData.category ? <p>{this.props.heroData.category}</p> : ''}
```

Comment:
I think they just want to completely remove the "category" from the hero? Maybe @courtneymcgee or @wlla can clarify. If so, we wouldn't need this property in the config.

Reply:
Um, I guess I wasn't sure if this was being removed _permanently_ (i.e. for future Staff Picks also). So I thought the best thing to do for now was to make the display determined by config. But I'm okay with removing it too!

Reply:
I honestly have no clue what the Hero content will contain. Everything else is good so let's merge and update later once we have better requirements.